Posit Infrastructure/FAQs.md

```diff
@@ -111,6 +112,12 @@ install.packages("ranger")
 
 Some packages, e.g. `{xlsx}`, `{XLconnect}`, depend on an installation of [Java](https://en.wikipedia.org/wiki/Java_(software_platform)) in order to work.  There are no current or future plans to support Java on Posit Workbench.  Alternative packages such as `{openxlsx}` that do not rely on Java should be used instead.
 
+#### What should I do if I encounter a 'failed to lock directory' error?
+If you are trying to install an R package and see an error message in the console regarding locked directories (e.g. something like `00LOCK` or `failed to lock directory`), try using the `--no-lock` option with `install.packages()` as follows:
```

Comment:
Just a practice at leaving comments... This looks good and no typos as far as I can see!